src/compiler/transformers/test/convert-decorators.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { transpileModule } from './transpile';
```

Comment:
Can we add a few tests around `super()` please?

Reply:
oh yeah that's a good idea, will do